xmr-btc/src/serde.rs

```diff
@@ -0,0 +1,152 @@
+pub mod cross_curve_dleq_scalar {
```

Comment:
cross-curbe-dleq is our repo, right? It'd be good if this is pushed in the repo that defines the type (behind a feature flag) that defined in this (xmr-btc) repo.

Reply:
Just an FYI: it's done already.